### PR TITLE
MTTL-538 Refactoring

### PR DESCRIPTION
**Pull request by evak6979**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/MTTL-538

### *What is the problem we're trying to solve*

Refactor/Clearing code

### CODE REVIEW

File: PersonApi.Tests/Readme.md [hunk 1]
```diff
@@ -1,14 +1,29 @@
-# Running a local coverage report
```
Comment (mikemoody-amido) [inline PersonApi.Tests/Readme.md:1]: You updated the wrong readme. 
You should be updating the readme at the root of the repo, not this one on the tests project.
(Content was good though)

Comment (evak6979) [reply PersonApi.Tests/Readme.md:1]: :D figures I'd do something like that. I'll move it around. Thanks :)

File: PersonApi/readme.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Person Api
```
Comment (mikemoody-amido) [inline PersonApi/readme.md:1]: This should be the readme in the root of the repo (not a project)

Comment (evak6979) [reply PersonApi/readme.md]: All done

**Review by mikemoody-amido — APPROVED** [review]

**Pull request merged**